src/commands/api-mesh/__tests__/create.test.js

```diff
@@ -1758,4 +1761,74 @@ describe('create command tests', () => {
 		]
 	`);
 	});
+
+	test('should show prod edge mesh url on workspace named "Production" if feature is enabled', async () => {
+		// mock the edge mesh url feature to be enabled
+		getTenantFeatures.mockResolvedValue({
+			imsOrgId: selectedOrg.code,
```

Comment:
Did we intend to have selected org code be interchangeable with imsOrgId here? It was my understanding that org code and id were different and if so the test values should be distinct enough to catch that error.

Reply:
This has an unfortunate backstory in that we named the property `imsOrgId` in the database despite the value actually being the `imsOrgCode`. They should not be used interchangeably, but SMS understands the orgCode to be the orgId. We decided to continue the trend for the `TenantsFeature` table and thus the imsOrgId which SMS is returning (as it is named in DynamoDB) is mocked accordingly here.

Reply:
We could rename this property in the response object to `imsOrgCode`, but it would not match the field in `TenantsFeatures` table. We could subsequently rename the field in the `TenantsFeatures` table, but then it would not match the `imsOrgId` field name in the other DynamoDB tables. Either way, it is confusing 🙁

Reply:
cc: @brasewel @dthampy

Reply:
although @amolina-adobe and I discussed this during this ticket's implementation, we didn't reach a decision as to we should fix this inconsistency in this ticket, since it would be a much bigger effort

Reply:
Yeah I think let's hold off on making drastic changes to rename things but yes we need to fix this eventually.